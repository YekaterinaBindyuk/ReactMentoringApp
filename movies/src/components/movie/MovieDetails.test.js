import React from "react";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const releaseDate = "01.01.2000";
  const duration = 180;
  const timeUnit = "min";
  const durationToRender = duration + " " + timeUnit;
  const tree = renderer
    .create(
      <div>
        <span className="mr-5">Release date: {releaseDate}</span>
        <span className="ml-5 badge badge-pill badge-dark">
          {durationToRender}
        </span>
      </div>
    )
    .toJSON();
  expect(tree).toMatchSnapshot(releaseDate, duration);
});
