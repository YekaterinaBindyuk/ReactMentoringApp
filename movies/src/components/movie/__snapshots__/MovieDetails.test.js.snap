// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: 01.01.2000 1`] = `
<div>
  <span
    className="mr-5"
  >
    Release date: 
    01.01.2000
  </span>
  <span
    className="ml-5 badge badge-pill badge-dark"
  >
    180 min
  </span>
</div>
`;
